src/main/java/cloud/fogbow/ras/core/models/orders/Order.java

```diff
@@ -66,6 +67,18 @@
     @Transient
     private SystemUser systemUser;
 
+    @Column
+    @Size(max = FIELDS_MAX_SIZE)
+    private String userId;
+
+    @Column
+    @Size(max = 1024)
```

Comment:
Will 1024 be enough? Please, have a look at the already defined SystemUser child classes to have an idea whether this size is adequate. Also, it would be good to add a comment there to warn developers that classes that extend SystemUser need to be aware of this size limit.